### ISSUE TITLE
Should we archive this repo?

### ISSUE THREAD

**Issue by trishankatdatadog** [uptane/obsolete-reference-implementation#205]
Should we mark this repository as archived? It has long since not seen active development, and is probably not the best reference implementation for various reasons.

**Comment by lukpueh** [comment]
> It has long since not seen active development

True. It also pins some outdated dependencies and a stale TUF fork hosted by @awwad.

>  not the best reference implementation for various reasons. 

Can't tell. Do you have any particular reasons in mind? The demo instructions seem quite elaborate to me.

> Should we mark this repository as archived?

I think we should. As per the [GitHub docs on archiving](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-archiving-repositories) the repo will remain visible and available in read-only mode, so visitors can still use it for educational purposes.

If wished, I can patch the README to say that this repo currently is not under active development (but may still be used as a guide to learn the standard?). cc CB @JustinCappos and co-maintainers @mnm678, @SantiagoTorres, @awwad.

**Comment by JustinCappos** [comment]
I'm supportive of this.

**Comment by mnm678** [comment]
I agree. This repo is probably already missing newer changes to Uptane, so achiving it will make it more clear that this is an implementation of an older version of the standard.

**Comment by lukpueh** [comment]
See https://github.com/uptane/uptane/pull/206 for the mentioned README patch.
